addons/mail/static/tests/discuss/core/web/command_palette.test.js

```diff
@@ -8,25 +8,23 @@ import {
     triggerHotkey,
 } from "@mail/../tests/mail_test_helpers";
 import { describe, test } from "@odoo/hoot";
-import { registry } from "@web/core/registry";
-import { advanceTime } from "@odoo/hoot-mock";
 import { Command, serverState } from "@web/../tests/web_test_helpers";
 
 describe.current.tags("desktop");
 defineMailModels();
 
-const commandSetupRegistry = registry.category("command_setup");
-
-test("open the chatWindow of a user from the command palette", async () => {
+test("can open DM from @username in command palette", async () => {
+    const pyEnv = await startServer();
+    pyEnv["res.partner"].create({ name: "Mario" });
     await start();
     triggerHotkey("control+k");
     await insertText(".o_command_palette_search input", "@");
-    await contains(".o_command", { count: 2 });
```

Comment:
Keep all the count in these tests, they help a lot when debugging and to catch unexpected additions.